README.md

```diff
@@ -23,10 +23,49 @@ tar -xvzf multipy_runtime.tar.gz
 In order to run PyTorch models, we need to link to libtorch (PyTorch's C++ distribution) which is provided when you [pip or conda install pytorch](https://pytorch.org/).
 If you're not sure which ABI value to use, it's important to note that the pytorch C++ binaries, provided when you pip or conda install, are compiled with an ABI value of 0. If you're using libtorch from the pip or conda distribution of pytorch then ensure to use multipy installation with an ABI of 0 (`nightly-runtime-abi-0`).
```

Comment:
I need to think a bit more about that -- didn't want to update that until we fixed the nightlies. This is all built with ABI=0 since that's what Python requires

Reply:
Yeah we need to test this stuff more properly. I'll write an issue to test this stuff.